lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -211,6 +215,14 @@ check_resource_details_key_and_value:
   element: &key_value
     selector:
       xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')]
+check_key_and_resource_link:
+  element:
+    selector:
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//a[contains(@href,'<resource_link>')]
+check_key_and_action_link:
+  element:
+    selector:
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button
```

Comment:
![tolerations](https://user-images.githubusercontent.com/12692381/64511486-05693400-d317-11e9-8a21-66e128ab16e7.png)
Used to check actions like this